Makefile

```diff
@@ -176,24 +183,16 @@ help:
 	@echo
 
 
+
+
 # TODO: add targets `translate` when merged
 .PHONY: all
 all: setup config-templates lint build
 
 
 .PHONY: setup
-setup: $(VENV) $(NODE_MODULES)
-
-
-requirements-py3.txt:
-	@echo "${GREEN}requirements-py3.txt has changed...${RESET}";
-
-
-# TODO: replace through pipenv as in the other projects.
-$(VENV): requirements-py3.txt
-		test -d "$(VENV)" ||  ( $(SYSTEM_PYTHON_CMD) -m venv $(VENV) && \
-		${PIP} install $(PIP_QUIET) --upgrade pip==21.2.4 setuptools --index-url ${PYPI_URL}  && \
-		${PIP} install $(PIP_QUIET) -r requirements-py3.txt --index-url ${PYPI_URL}  -e . )
+setup: $(NODE_MODULES)
+	if ! [ -d "$(VENV)" ] ;then pipenv install && pipenv install --dev;fi
```

Comment:
`pipenv install --dev` is sufficient here no need to do a `pipenv install` as well.